src/axom/primal/operators/evaluate_integral.hpp

```diff
@@ -99,6 +99,61 @@ double evaluate_scalar_line_integral(const primal::BezierCurve<T, NDIMS>& c,
   return detail::evaluate_scalar_line_integral_component(c, scalar_integrand, quad);
 }
 
+/*!
+ * \brief Evaluate a line integral on a single NURBS curve on a scalar field
+ *
+ * \param [in] n the NURBS curve object
+ * \param [in] scalar_integrand the lambda function representing the integrand. 
+ * Must accept a Point<T, NDIMS> as input, and return a double.
+ * \param [in] npts the number of quadrature nodes
```

Comment:
We should probably mention that we do Bezier extraction, so it's `npts` quadrature points per knot span of the NURBS curve.

Reply:
Fixed here and elsewhere